smart_contracts.rst

```diff
@@ -153,10 +154,12 @@ Allows a channel participant to close the channel. The channel cannot be settled
     event ChannelClosed(uint channel_identifier, address closing_participant);
 
 - ``channel_identifier``: Channel identifier assigned by the current contract.
+- ``balance_hash``: Hash of the balance data ``keccak256(transferred_amount, locked_amount, locksroot)``
 - ``nonce``: Strictly monotonic value used to order transfers.
+- ``additional_hash``: Computed from the message. Used for message authentication.
 - ``transferred_amount``: The monotonically increasing counter of the partner's amount of tokens sent.
+- ``locked_amount``: The sum of the all the tokens that correspond to the locks (pending transfers) contained in the merkle tree.
```

Comment:
Shouldn't the descriptions of `transferred_amount` and `locked_amont` have moved?

Reply:
They will move in another PR. I opened an issue about this here: https://github.com/raiden-network/spec/issues/57.